capa/features/extractors/viv/extractor.py

```diff
@@ -20,21 +20,22 @@
 from capa.features.common import Feature
 from capa.features.address import Address, AbsoluteVirtualAddress
 from capa.features.extractors.base_extractor import BBHandle, InsnHandle, FunctionHandle, FeatureExtractor
-
+from capa.features.extractors.strings import DEFAULT_STRING_LENGTH
 logger = logging.getLogger(__name__)
 
 
 class VivisectFeatureExtractor(FeatureExtractor):
-    def __init__(self, vw, path, os):
+    def __init__(self, vw, path, os, len):
         super().__init__()
         self.vw = vw
         self.path = path
+        self.len = len
         with open(self.path, "rb") as f:
             self.buf = f.read()
 
         # pre-compute these because we'll yield them at *every* scope.
         self.global_features: List[Tuple[Feature, Address]] = []
-        self.global_features.extend(capa.features.extractors.viv.file.extract_file_format(self.buf))
+        self.global_features.extend(capa.features.extractors.viv.file.extract_file_format(self.buf, self.len))
```

Comment:
why does extracting the file format need the string length configuration? can we remove this kwarg?